packages/components/src/raw-icon/index.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
👍 Nice little component. Why not just `Icon`, Thoughts on the name @aduth @mcsf

Reply:
My expectations from the name and description don't really follow into the implementations. If it's "without styles", why do we apply a default width? And an unstyled variation seems more appropriate as a prop of a default component type, e.g. `<Icon isUnstyled />`.

Reply:
Well, I consider the width an attribute, and not a style 🤷‍♂️ 

And regarding the component name - `Icon` felt a little too broad, and I'd probably expect an icon component to be more flexible in terms of how the icon could _look_. So I'd instead expect to have an `Icon` component that could use the `RawIcon`, but add on top of it. 

What do y'all think?

Reply:
I'm good with changing to `UnstyledIcon`, @aduth @youknowriad does that make sense to you?

Reply:
>I'd probably expect an icon component to be more flexible in terms of how the icon could look.

Can you elaborate on what you mean by this? I'm not sure I follow, and am having a hard time understanding what specifically we're implying in these multiple layers of icon components.

Reply:
I personally still have a small preference for just  `Icon`, if we need to make a distinction between styled and unstyled later, we can add it as a prop.

Reply:
I've been thinking and started gathering a list of what `<Icon />` would provide more than `<UnstyledIcon />`, but I didn't get too far with it. 

I guess I'm good with renaming this one to `<Icon />`, it seems @mcsf and @youknowriad are good with `<Icon />`, @aduth does that sound well to you?

Reply:
Works for me. 👍

Reply:
Cool, thanks 👍 

Renamed `<RawIcon />` to `<Icon />` in https://github.com/WordPress/gutenberg/pull/10651/commits/3486d0bcc6d2926f453e7b0586e378f93d2c4562

I'd appreciate another look when y'all get a chance.